### DISCUSSION TITLE
How to train the agents by using HER （hindsight experience replay，HER）?

### DISCUSSION THREAD

**Discussion by beiketaoerge** [Toni-SM/skrl#150] [Q&A]
In stable-baseline3,we can use the code    `model = DDPG("MultiInputPolicy", env,replay_buffer_class=HerReplayBuffer, verbose=1, tensorboard_log=logs_dir,batch_size=256,device="cuda")` to achieve HER.If we use the code   `memories = RandomMemory(memory_size=15000,  device=device, `replacement=False)` in skrl, can we achieve HER? Thank you for your answer.
`

**Comment by Toni-SM** [comment]
Hi @beiketaoerge 

Sorry for late response.
Unfortunately, **skrl** doesn't implement HER for the time being.
It is in the roadmap but I have not found the time for it :see_no_evil:
